.circleci/config.yml

```diff
@@ -166,6 +150,51 @@ jobs:
           working_directory: tests/xcc-a
           command: cargo build --target wasm32-wasi --bins && cargo test --lib
 
+  docker_build:
+    executor: docker_builder
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build Docker image
+          command: docker build -t $IMAGE_NAME:latest .
+      - run:
+          name: Archive Docker image
+          command: docker save -o image.tar $IMAGE_NAME
```

Comment:
do you want to push this to docker hub as well?

Reply:
not yet. it needs to check that matle-build builds first. It'll push to `pending` and then, once that succeeds, it'll push to `latest`.